src/DynamoCore/Utilities/Configurations.cs

```diff
@@ -145,7 +145,7 @@ public class Configurations
 
         #region CodeBlockNode
 
-        public static readonly double CodeBlockPortHeightInPixels = 17.563333333333336;
+        public static readonly double CodeBlockPortHeightInPixels = 17.573333333333336;
```

Comment:
The original number still result in tiny bit of difference, changing this so that they lined up nicer than with the old number.